api/src/main/webapp/WEB-INF/cron_test.yaml

```diff
@@ -8,5 +8,10 @@ cron:
   schedule: every 24 hours
   timezone: UTC
   target: api
+- description: Re-enqueues any egress events which have been pending for too long.
+  url: /v1/cron/checkPendingEgressEvents
+  schedule: 31 dec 2024 23:59
```

Comment:
can't fully disable as far as I know, so I set it to run once a long time in the future